waiter/bin/ci/minimesos

```diff
@@ -2,9 +2,9 @@
 
 set -e
 
-MINIMESOS_TAG="0.13.0"
+MINIMESOS_TAG="0.13.2"
 PARAMS="$@"
-MINIMESOS_CLI_IMAGE="containersol/minimesos-cli"
+MINIMESOS_CLI_IMAGE="daowen/minimesos-cli"
```

Comment:
Should we use the `twosigma` one?

Reply:
Now that we have a twosigma version, definitely! Fixed.